usaspending_api/awards/v2/filters/filter_helpers.py

```diff
@@ -211,18 +156,16 @@ def can_use_month_aggregation(time_period):
 
 
 def can_use_total_obligation_enum(amount_obj):
-    bins = []
     try:
         for v in amount_obj:
             lower_bound = v.get("lower_bound")
             upper_bound = v.get("upper_bound")
-            for key, limits in WEBSITE_AWARD_BINS.items():
-                if lower_bound == limits["lower"] and upper_bound == limits["upper"]:
-                    bins.append(key)
+            for key, values in WEBSITE_AWARD_BINS.items():
+                if lower_bound == values["lower"] and upper_bound == values["upper"]:
                     break
-
-        if len(bins) == len(amount_obj):
-            return True
+            else:
```

Comment:
Is the `else` suppose to be indented one more?

Reply:
No, this is a `for...else`. If it was indented again to the `if` then only the first enum value would be examined